src/bootstrap/src/lib.rs

```diff
@@ -30,26 +33,20 @@ use build_helper::ci::gha;
 use build_helper::exit;
 use sha2::digest::Digest;
 use termcolor::{ColorChoice, StandardStream, WriteColor};
-use utils::channel::GitInfo;
-use utils::helpers::hex_encode;
 
+pub use self::core::builder::PathSet;
+pub use self::core::config::Config;
+pub use self::core::config::flags::{Flags, Subcommand};
 use crate::core::builder;
 use crate::core::builder::{Builder, Kind};
 use crate::core::config::{DryRun, LldMode, LlvmLibunwind, Target, TargetSelection, flags};
+pub use crate::utils::change_tracker::{
```

Comment:
IMO previous approach was easier to track things. Right now it's not immediately clear which items are being re-exported from this module since they are mixed in with regular `use` declarations.

Reply:
I'll revert the changes to lib.rs, I don't have strong preferences here.

Reply:
Dropped the `lib.rs` import changes.